packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -286,6 +286,7 @@ class Dropdown extends React.Component {
                                 icon={this.props.is_alignment_left ? 'IcChevronLeft' : 'IcChevronDown'}
                                 className={classNames('dc-dropdown__select-arrow', {
                                     'dc-dropdown__select-arrow--left': this.props.is_alignment_left,
+                                    'dc-dropdown__select-arrow--show': this.state.is_list_visible,
```

Comment:
Can we change this to `--upward`? `show` is way too general for this.

Reply:
actually we don't have --upward, but we have --up. can we use that instead?